sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/README.md

```diff
@@ -24,13 +24,13 @@ ServiceClient serviceClient = ServiceClient.CreateFromConnectionString(connectio
 
 ### Install the package
 
-Install the Video Analyzer Edge client library for .NET with NuGet:
+Install the Video Analyzer Edge client library for .NET with [NuGet](https://www.nuget.org/packages/Azure.Media.VideoAnalyzer.Edge):
```

Comment:
The link should be to just https://nuget.org, as with other READMEs. The package name is in the sample command that follows.